typescript/infra/src/utils/timelock.ts

```diff
@@ -71,6 +78,26 @@ export async function timelockConfigMatches({
         `Proposers missing role for ${chain} at ${address}: ${proposersMissing.join(', ')}`,
       );
     }
+
+    // Ensure the cancellers have the CANCELLER_ROLE
```

Comment:
If we have cancellers defined, we'll want to check both that the expected cancellers have the role and also that other proposers _do not_ have the role